hasura/metadata/databases/default/tables/public_circle_api_keys.yaml

```diff
@@ -2,6 +2,45 @@ table:
   name: circle_api_keys
   schema: public
 object_relationships:
+- name: circle
+  using:
+    foreign_key_constraint_on: circle_id
 - name: createdByUser
   using:
     foreign_key_constraint_on: created_by
+select_permissions:
+- permission:
+    columns:
+    - circle_id
+    - created_by
+    - create_vouches
+    - read_circle
+    - read_epochs
+    - read_member_profiles
+    - read_nominees
+    - read_pending_token_gifts
+    - update_circle
+    - update_pending_token_gifts
+    - hash
+    - name
+    - created_at
```

Comment:
if we're exposing `created_at` maybe also expose `updated_at` ?

Reply:
They should never be updated, just created or deleted. If a user needs to change scope for an API key, they should regenerate it.

Maybe in the future we could allow changing scopes for an "installed integration" when we setup the flow for verified 3rd party apps, but with this API key setup delete + create only is best practice. (e.g. see Github and generating personal access tokens)